docs/src/main/sphinx/admin/event-listeners.rst

```diff
@@ -0,0 +1,8 @@
+===============
```

Comment:
I think you misunderstood ... move this content into the admin.rst file..  with different header usage..